docs/reference.md

```diff
@@ -333,6 +333,11 @@ $ curl -u USERNAME:PASSWORD https://openwhisk.ng.bluemix.net/api/v1/namespaces/w
 ]
 ```
 
+The API of OpenWhisk also supports asynchronous calls on websites from the client.
+To allow these calls on the client side, we respond to OPTIONS calls with a list of allowed Headers and allowed Hosts.
```

Comment:
To allow these calls from the browser directly, OpenWhisk responds to `OPTIONS` calls with  *Cross-Origin Resource Sharing* headers for allowed origins and headers.